Mimecast v2 - Migrate Command (#45292)
demisto/content: Packs/Mimecast/ReleaseNotes/3_1_0.md

#### Integrations

##### Mimecast v2

- Improved implementation of OAuth 2.0 authentication.
- Updated the following commands to the Mimecast v2 API:
  - ***mimecast-create-block-sender-policy***
  - ***mimecast-update-block-sender-policy***
  - ***mimecast-list-policies***
  - ***mimecast-get-policy***
  - ***mimecast-delete-policy***

> - Added support for the following arguments in the **mimecast-create-block-sender-policy** command:

Reviewer: ```suggestion
- Added support for the following arguments in the ***mimecast-create-block-sender-policy*** and ***mimecast-update-block-sender-policy*** commands:
```

  - *from_date*
  - *to_date*
  - *from_attribute_id*
  - *to_attribute_id*
  - *from_attribute_value*
  - *to_attribute_value*

> - Added support for the following arguments in the **mimecast-update-block-sender-policy** command:

Reviewer: ```suggestion
```

>   - *from_date*

Reviewer: ```suggestion
```

>   - *to_date*

Reviewer: ```suggestion
```

>   - *from_attribute_id*

Reviewer: ```suggestion
```

>   - *to_attribute_id*

Reviewer: ```suggestion
```

>   - *from_attribute_value*

Reviewer: ```suggestion
```

>   - *to_attribute_value*

Reviewer: ```suggestion
```

> - Added support for the *next_token* argument in the **mimecast-list-policies** command.

Reviewer: ```suggestion
- Added support for the *next_token* argument in the ***mimecast-list-policies*** command.
```

- Added the new option *address_attribute_value* to the *fromType* argument in the ***mimecast-create-block-sender-policy*** command.

> - Added the new option *address_attribute_value* to the *tomType* argument in the ***mimecast-update-block-sender-policy*** command.

Reviewer: ```suggestion
- Added the new option *address_attribute_value* to the *toType* argument in the ***mimecast-update-block-sender-policy*** command.
```

- Updated the Docker image to: *demisto/python3:3.12.13.12042988*.

**Breaking Changes:**

> - The context outputs on the following commands have changed:

Reviewer: ```suggestion
- The context outputs for the following commands have changed:
```

  - ***mimecast-create-block-sender-policy***
  - ***mimecast-update-block-sender-policy***
  - ***mimecast-list-policies***
  - ***mimecast-get-policy***
  - ***mimecast-delete-policy***

> - Updated the *fromType* argument in the ***mimecast-update-block-sender-policy*** command, it no longer accepts *free_mail_domains* or *header_display_name*.

Reviewer: ```suggestion
- Updated the *fromType* argument in the ***mimecast-update-block-sender-policy*** command to no longer accept *free_mail_domains* or *header_display_name*.
```

> - Updated the **fromPart** argument to be required in the **mimecast-create-block-sender-policy** command.

Reviewer: ```suggestion
- Updated the *fromPart* argument to be required in the ***mimecast-create-block-sender-policy*** command.
```
